{{cookiecutter.project_slug}}/README.md

```diff
@@ -0,0 +1,17 @@
+## Using Docker
+
+1. [Install Docker](https://docs.docker.com/get-docker/) on your machine
+2. `docker-compose up --build --force-recreate web`
+3. It's now available at: http://localhost:{{cookiecutter.port}}/
+
+Migrations are run automatically at startup, and suppliers are added automatically at startup
+
+
+## Running tests
+
+    make test
+
+
+## Checking code
+
+    make check-python-code
```

Comment:
Add some stuff on using the cookiecutter, which as I recall was something like installing python package, cloning this repo, running some command like `cookiecutter cookie-cutter`.